calculator-app/script.js

```
@@ -0,0 +1,21 @@
+let display = document.getElementById("display");
+
+function press(val) {
+  display.value += val;
+}
+
+function calculate() {
+  try {
+    display.value = eval(display.value); // (ok for practice)
```

Biome (2.4.10)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)